Switched to branch 'candidate'
Client-Request-ID=fdb24155-6dfd-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Tue, 12 Jun 2018 05:03:49 GMT, Server-Request-ID=f3cf9ad6-c01e-0045-630a-0289a0000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:f3cf9ad6-c01e-0045-630a-0289a0000000Time:2018-06-12T05:03:49.1248329Z</Message></Error>.
Client-Request-ID=fdb24157-6dfd-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Tue, 12 Jun 2018 05:03:48 GMT, Server-Request-ID=6c1d7142-a01e-0092-470a-02c387000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:6c1d7142-a01e-0092-470a-02c387000000Time:2018-06-12T05:03:49.3312837Z</Message></Error>.
Client-Request-ID=fdb24159-6dfd-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Tue, 12 Jun 2018 05:03:48 GMT, Server-Request-ID=45f9b57f-601e-008f-190a-021a6d000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:45f9b57f-601e-008f-190a-021a6d000000Time:2018-06-12T05:03:49.4919758Z</Message></Error>.
Client-Request-ID=fdb2415b-6dfd-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Tue, 12 Jun 2018 05:03:49 GMT, Server-Request-ID=30664ee7-c01e-0028-020a-02238e000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:30664ee7-c01e-0028-020a-02238e000000Time:2018-06-12T05:03:49.6513706Z</Message></Error>.
Traceback (most recent call last):
  File "./02.local.py", line 92, in <module>
    min(metrics['mse']), 
KeyError: 'mse'
Client-Request-ID=a892b5a5-6e03-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Tue, 12 Jun 2018 05:44:23 GMT, Server-Request-ID=d10d47b1-a01e-001a-0410-027b5e000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:d10d47b1-a01e-001a-0410-027b5e000000Time:2018-06-12T05:44:23.8974455Z</Message></Error>.
Client-Request-ID=a892b5a7-6e03-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Tue, 12 Jun 2018 05:44:23 GMT, Server-Request-ID=b9e06bf7-f01e-002b-7210-022089000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:b9e06bf7-f01e-002b-7210-022089000000Time:2018-06-12T05:44:24.0850291Z</Message></Error>.
Client-Request-ID=a968d7c5-6e03-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Tue, 12 Jun 2018 05:44:24 GMT, Server-Request-ID=2eef559b-d01e-003c-3f10-02e0ea000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:2eef559b-d01e-003c-3f10-02e0ea000000Time:2018-06-12T05:44:24.2930500Z</Message></Error>.
Client-Request-ID=a968d7c7-6e03-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Tue, 12 Jun 2018 05:44:24 GMT, Server-Request-ID=74c463a7-d01e-00d2-7910-02ea69000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:74c463a7-d01e-00d2-7910-02ea69000000Time:2018-06-12T05:44:24.4849197Z</Message></Error>.
Traceback (most recent call last):
  File "./10.o16n.py", line 88, in <module>
    model.delete()
  File "/home/ninghai/miniconda3/envs/candidate/lib/python3.6/site-packages/azureml/core/model.py", line 313, in delete
    'Content: {}'.format(resp.status_code, resp.headers, resp.content))
azureml.exceptions.azureml_exception.WebserviceException: Received bad response from Model Management Service:
Response Code: 412
Headers: {'Date': 'Tue, 12 Jun 2018 05:51:37 GMT', 'Content-Type': 'application/json', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'api-supported-versions': '2018-03-01-preview', 'x-ms-client-request-id': '48e2e29099f64d11a63b9441e8774781', 'x-ms-client-session-id': '', 'Strict-Transport-Security': 'max-age=15724800; includeSubDomains; preload'}
Content: b'{"code":"PreconditionFailed","statusCode":412,"message":"PreconditionFailed","details":[{"code":"DeletionRequired","message":"The model with id best_model.pkl:189 can not be deleted because it is currently being used in one or more images"}]}'
Switched to branch 'master'
